tests/BasicSearchTests.FunctionalTests.Core/BaseConfig.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Net.Http;
+using BasicSearchTests.FunctionalTests.Core.TestSupport;
+using System.Net;
+
+namespace BasicSearchTests.FunctionalTests.Core
+{
+    public class BaseConfig
+    {
+        protected HttpClient Client;
+        protected RetryHandler RetryHandler;
+
+        public BaseConfig()
+        {
+            // Arrange
+            IgnoreBadCertificates();
+            RetryHandler = new RetryHandler(new HttpClientHandler());
+            Client = new HttpClient(RetryHandler) { BaseAddress = new Uri(EnvironmentSettings.SearchServiceBaseUrl) };
+        }
+
+        /// <summary>
+        /// This method causes to bypass errors caused by SLL-Errors.
+        /// </summary>
+        private static void IgnoreBadCertificates()
```

Comment:
I think you can just remove this function and put this line in the constructor and add the `Bypass errors caused by SLL-Errors` comment above it.